lightning/src/ln/channelmanager.rs

```diff
@@ -3471,11 +3473,11 @@ impl<ChanSigner: ChannelKeys, M: Deref + Sync + Send, T: Deref + Sync + Send, K:
 		if msg.channel_id == [0; 32] {
 			for chan in self.list_channels() {
 				if chan.remote_network_id == *counterparty_node_id {
-					self.force_close_channel(&chan.channel_id);
+					self.force_close_channel(&chan.channel_id).unwrap();
 				}
 			}
 		} else {
-			self.force_close_channel(&msg.channel_id);
+			self.force_close_channel(&msg.channel_id).unwrap();
```

Comment:
This handles an untrusted message from a peer - if they send us a garbage channel_id we shouldn't crash.

Reply:
Ups, didn't realize that came from a peer.

Rebased in 03a72f1